erpnext/manufacturing/doctype/bom/test_bom.py

```diff
@@ -147,8 +147,8 @@ def reset_item_valuation_rate(item_code, warehouse_list=None, qty=None, rate=Non
 
 	if not warehouse_list:
 		warehouse_list = frappe.db.sql_list("""
-			select warehouse from `tabBin`
-			where item_code=%s and actual_qty > 0 and company = '_Test Company'
+			select tabBin.warehouse, tabWarehouse.company from `tabBin`
+			INNER JOIN tabWarehouse ON tabBin.name=tabWarehouse.name  where item_code=%s and actual_qty > 0;
```

Comment:
The query is wrong, `name` is not the common column in both of the tables, please check and rework on this.